coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java

```diff
@@ -544,6 +545,19 @@ public boolean areActiveQuestionnaires(List<AnswerHeader> answerHeaders) {
         return false;
     }
 
+    public boolean areAnsweredQuestionnaires(List<AnswerHeader> answerHeaders) {
+        for (AnswerHeader answerHeader : answerHeaders) {
+            if (answerHeader.isActive()) {
+                for (Answer answer : answerHeader.getAnswers()) {
+                    if (StringUtils.isNotEmpty(answer.getAnswer())) {
```

Comment:
per jira comment "If you can discern that any questions are answered but the questionnaire is not complete, then by all means, include the option to copy the questionnaire"